internal/rtutils/rtutils.go

```diff
@@ -4,10 +4,22 @@ import (
 	"fmt"
 	"runtime"
 	"time"
-
-	"github.com/hashicorp/go-multierror"
 )
 
+type packedErrors struct {
```

Comment:
It might be worth adding a comment that this was duplicated in another package in order to avoid an import cycle, just like the previous instance.